### PR TITLE
Issue#818

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added Image plus text adding the new *Enable Mailbox Intelligence based impersonation protection* issue #818

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/set-up-anti-phishing-policies.md [hunk 1]
```diff
@@ -82,11 +83,16 @@ As you set up or edit your ATP anti-phishing policies, you can choose from sever
...
+|**Enable mailbox intelligence** <br/> |Enables or disables mailbox intelligence for this policy. You can only enable mailbox intelligence for cloud-based accounts, that is, accounts whose mailbox is hosted entirely in Office 365. <br/> | Mailbox intelligence is built around the people you send and receive mail from both inside the organization and outside the organization. <br/> |
+|**Enable mailbox intelligence based impersonation protection** <br/> |Enables or disables mailbox intelligence for impersonation protection for this policy. The important here is the control of the impersonation for a given mailbox. <br/> |When you want to enhance impersonation results for users based on each user's individual sender map. This intelligence allows Office 365 to customize user impersonation detection and better handle false positives. When user impersonation is detected, based on mailbox intelligence, you can define what action to take on the message. <br/> |
```
Comment (JohanFreelancer9) [inline SecurityCompliance/set-up-anti-phishing-policies.md]:
```suggestion
|**Enable mailbox intelligence based impersonation protection** <br/> |Enables or disables mailbox intelligence for impersonation protection for this policy. The important consideration here is the control of the impersonation for a given mailbox. <br/> |When you want to enhance impersonation results for users based on each user's individual sender map. This intelligence allows Office 365 to customize user impersonation detection and better handle false positives. When user impersonation is detected, based on mailbox intelligence, you can define what action to take on the message. <br/> |
```

File: SecurityCompliance/set-up-anti-phishing-policies.md [hunk 2]
```diff
@@ -82,11 +83,16 @@ As you set up or edit your ATP anti-phishing policies, you can choose from sever
...
 
+> [!IMPORTANT]
+> ![Image](media/EnableMailboxIntelligenceBasedImpersonation.png)
+> New Important settings, separting **Mailbox Intelligence" from "Mailbox Intelligence based impersonation protection". Impersonation is when you programatically access your mailbox and execute actions over the mailbox using an script, API or a 3rd party software, or even malicious code under your mailbox.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/set-up-anti-phishing-policies.md]:
```suggestion
> New Important settings, separating **mailbox intelligence" from "mailbox intelligence based impersonation protection". Impersonation is when you programmatically access your mailbox and execute actions over the mailbox using a script, API, third party software, or even malicious code under your mailbox.
```

**Comment by scanum** [comment]
@MSFTTracyP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**